CONTRIBUTING.md

```diff
@@ -21,6 +21,14 @@ phpize > /dev/null && \
 make clean > /dev/null && make all > /dev/null && make install
 ```
 
+To verify that the installation was successful, run the following command which indicates the version of the installed
+extension. It must end with `-dev` suffix.
```

Comment:
```suggestion
To verify that the installation was successful, run the following command, which
will report `phpinfo()` output for the extension:
```

A few changes here:

 * Applying 80-character wrap for consistency with other lines
 * Added comma before "which" (see: [grammar rules](https://www.grammarly.com/blog/comma-before-which/))
 * Referred to `phpinfo()` since that's directly what `--ri` reports. We can optionally added "(including its version)" but I don't think it's necessary for this audience.
 * Even before the previous change, I was going to suggest removing the bit about the `-dev` suffix as these are general rules for building from source. Even though the shell example above starts with a fresh git clone, these are the same instructions one might follow when building a tag from source. This is basically a local copy of [Building the MongoDB PHP Driver from source ](https://www.php.net/manual/en/mongodb.installation.manual.php) in the PHP.net docs.